internals/getCsp.js

```diff
@@ -3,8 +3,15 @@
 // - remove 'unsafe-inline' style by precomputing theme hash
 // - add report-uri to gather errors if anything was missed
 
+const cspEnhancement = `
+  frame-ancestors 
+    'self' 
+    https: http://localhost:* http://127.0.0.1:*;
+  `
```

Comment:
I wouldn't call it enhancement :D

Reply:
omg I forgot to push commit before merging. Anyway will change the variable name in a next extension PR.